cinema/views.py

```diff
@@ -56,3 +124,24 @@ def get_serializer_class(self):
             return MovieSessionDetailSerializer
 
         return MovieSessionSerializer
+
+
+class OrderViewSet(viewsets.ModelViewSet):
+    queryset = Order.objects.all()
```

Comment:
you can remove this since you always override queryset in method below, OR
you can remove get_queryset() method and assign it here: 
queryset = Order.objects.prefetch_related(
            "tickets__movie_session__movie",
            "tickets__movie_session__cinema_hall"
        ).filter(user=self.request.user)